src/app/tool-editor/sections/arguments/argument-inspector.component.ts

```diff
@@ -0,0 +1,83 @@
+import {Subject} from "rxjs";
+import {Component, Input, Output} from "@angular/core";
+import {FormBuilder, FormGroup, FormControl} from "@angular/forms";
+import {CommandArgumentModel} from "cwlts/models/d2sb";
+import {ComponentBase} from "../../../components/common/component-base";
+
+@Component({
+    selector: "ct-argument-inspector",
+    template: `
+<ct-form-panel [borderless]="true" [collapsed]="false">
+
+        <span class="tc-header">
+            Basic
+        </span>
+        
+        <div class="tc-body">
+            <form [formGroup]="form">
+            
+                <!--Prefix Field-->
+                <div class="form-group">
+                    <label>Prefix:</label>
+                    <input type="text" class="form-control" [formControl]="form.controls['prefix']">
+                </div>
+                
+                <!--Expression Field-->
+                <div class="form-group">
+                    <label>Expression:</label>
+                    <ct-expression-input 
+                    [context]="context"
+                    [formControl]="form.controls['valueFrom']">                    
+                    </ct-expression-input>
+                </div>
+                
+                <!--Separator field-->            
+                <div class="form-group flex-container">
+                    <label>Separator:</label>
+                    <span class="align-right">
+                        {{form._value.separate? "Yes" : "No"}}
+                        <toggle-slider [formControl]="form.controls['separate']"></toggle-slider>
```

Comment:
`toggle-slider` has two new inputs, `[on]` and `[off]`, where you can pass in string for on and off state. Also, I changed the label and text for separator in the inputBinding, copy that one.